easybuild/tools/github.py

```diff
@@ -754,3 +966,73 @@ def __init__(self, user):
 def fetch_github_token(user):
     """Fetch GitHub token for specified user from keyring."""
     return GithubToken(user).token
+
+
+def install_github_token(token, github_user, silent=False):
+    """
+    Install specified GitHub token for specified user.
+
+    @param token: GitHub token to install
+    @param github_user: GitHub user to install token for
+    """
+    if github_user is None:
+        raise EasyBuildError("GitHub user must be specified to install GitHub token")
+
+    if HAVE_KEYRING:
+        # check if there's a token available already
+        current_token = fetch_github_token(github_user)
+
+        if current_token:
+            current_token = '%s..%s' % (current_token[:3], current_token[-3:])
```

Comment:
sure there is, may help with understanding why it doesn't allow authenticated access even though there's *some* value there

Say it prints this:

```
boe...gel
```

Then you probably know you switched token and username (or something like that)